### DISCUSSION TITLE
How to create auto labelling using custom REST request

### DISCUSSION THREAD

**Discussion by Anand195** [doccano/doccano#1867] [Q&A] [answered]
Hi, guys
I’m new here and found such a cool way to automatically annotate the text but thing is I don’t know much how to create custom REST request to generate auto annotation.

**Comment by david-engelmann** [comment, marked as answer]
@Anand195 [@Hironsan has a great write up in this pull request](https://github.com/doccano/doccano/pull/1997/commits/ee38de41053619c7ee177c45c2d070254de20693). You can find everything you need in [this documentation](https://github.com/doccano/doccano/blob/master/docs/advanced/auto_labelling_config.md)
